lib/consult/template.rb

```diff
@@ -76,20 +80,34 @@ def ordered_locations
 
     # Concatenate all the source templates together, in the order provided
     def contents
-      ordered_locations.map do |location|
+      @_contents ||= ordered_locations.map do |location|
         location.to_s.start_with?('consul') ? consul_contents(location) : disk_contents(location)
-      end.join
+      end.compact.join
     end
 
     def consul_contents(location)
       [@config[location]].compact.flatten.map do |key|
-        Diplomat::Kv.get(key, options: {}, not_found: :return, found: :return).force_encoding 'utf-8'
```

Comment:
I assume you had to remove the `not_found` and `found` args here in order to implement the rescue logic?

Reply:
Addressed in this commit message: https://github.com/veracross/consult/pull/48/commits/28220a8dc899b87d63b7f3c9ce78e6030d3bfaed

Reply:
thanks for the clarification